spark-3.3/clickhouse-spark/src/main/scala/org/apache/spark/sql/clickhouse/ClickHouseSQLConf.scala

```diff
@@ -201,4 +201,12 @@ object ClickHouseSQLConf {
       .version("0.8.0")
       .booleanConf
       .createWithDefault(false)
+
+  val FIXED_STRING_READ_AS: ConfigEntry[String] =
+    buildConf("spark.clickhouse.fixedStringReadAs")
+      .doc("read ClickHouse FixedString type as the specified Spark data type. Supported formats: binary, string")
+      .version("0.8.1")
```

Comment:
not sure what would be the proper version.

Reply:
0.8.0

Reply:
copy that.